### PR TITLE
Svg prop

### PR DESCRIPTION
**Pull request by kennetpostigo**
adds a svg property

### CODE REVIEW

File: src/ReactDOMRe.re [hunk 1]
```diff
@@ -67,6 +67,7 @@ external props :
     ~className: string=?, /* substitute for "class" */
     ~contentEditable: Js.boolean=?,
     ~contextMenu: string=?,
+    ~dataName: string=?
```
Comment (AgtLucas) [inline src/ReactDOMRe.re]: Missing a `,`

Comment (kennetpostigo) [reply src/ReactDOMRe.re]: thanks!

Comment (AgtLucas) [reply src/ReactDOMRe.re]: :)

**Comment by chenglou** [comment]
Thanks!

**Comment by chenglou** [comment]
Hold on, where is this attribute? https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes

**Comment by mobily** [comment]
hm 🤔 it's not valid SVG attribute

https://www.w3.org/TR/SVG/attindex.html
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute

**Comment by kennetpostigo** [comment]
My bad, not sure why sketch adds this when generating an SVG, when removed the svg renders properly

**Comment by chenglou** [comment]
Some random metadata I guess! Closing =)

**Pull request closed without merging**